### PR TITLE
Quick improvements

### PR DESCRIPTION
**Pull request by talsraviv**
Improvements in the wake of #235 and #241

### CODE REVIEW

**Comment by talsraviv** [comment]
[Still in progress]

File: duckduckhack/instant-answer-display/templates_reference.md [hunk 1]
```diff
@@ -748,19 +791,272 @@ templates: {
...
+------
+
+## `media_item_detail` Template
+
+### Template Diagram
+
+```
++------------------------------------------------------------------+
+						title, altSubtitle
+	image				subtitle
```
Comment (moollaza) [inline duckduckhack/instant-answer-display/templates_reference.md]: Looks like `subtitle` needs to be indented a little further

**Comment by moollaza** [comment]
@talsraviv should be good once the table is fixed!

File: duckduckhack/instant-answer-display/templates_overview.md [hunk 2]
```diff
@@ -446,39 +451,32 @@ When you specify this template group, it is equivalent to setting the properties
...
 
-### Example uses of the `media` template group
+### Example Uses
 
-- ["lord of the rings movie"](https://duckduckgo.com/?q=lord+of+the+rings+movie) ([code](https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/share/spice/movie/movie.js))
-    ![DuckDuckGo search for "lord of the rings movie"](https://images.duckduckgo.com/iu/?u=https%3A%2F%2Fraw.githubusercontent.com%2Fduckduckgo%2Fduckduckgo-documentation%2Fmaster%2Fduckduckhack%2Fassets%2Ftemplate_groups%2Flord_of_the_rings_movie.png&f=1)
-
-- ["movies"](https://duckduckgo.com/?q=movies) ([code](https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/share/spice/in_theaters/in_theaters.js))
-    ![DuckDuckGo search for "movies"](https://images.duckduckgo.com/iu/?u=https%3A%2F%2Fraw.githubusercontent.com%2Fduckduckgo%2Fduckduckgo-documentation%2Fmaster%2Fduckduckhack%2Fassets%2Ftemplate_groups%2Fmovies.png&f=1)
+*None - yet!*
```
Comment (moollaza) [inline duckduckhack/instant-answer-display/templates_overview.md]: FWIW we'll have one shortly. There's a task to migrate the Dogo News Spice to use the media group which will allow it to display images and text together :)

File: duckduckhack/instant-answer-display/templates_reference.md [hunk 3]
```diff
@@ -309,6 +347,47 @@ A template for drilling-down into a particular item on the same page. Emphasizes
...
+## `basic_icon_detail` Template
+
+A template for displaying textual information detail, with a small image, icon, or character badge.
+
+### Template Diagram
+
+```
++------------------------------------------------+
+image and/or badge      title
+                        subtitle/altSubtitle
```
Comment (moollaza) [inline duckduckhack/instant-answer-display/templates_reference.md]: actually `altSubtitle` should be below `subtitle` we allow you to display both

File: duckduckhack/instant-answer-display/templates_reference.md [hunk 4]
```diff
@@ -748,19 +837,273 @@ templates: {
...
+
+### Template Diagram
+
+```
++------------------------------------------------------------------+
+
+
+    [video content]             title
+                              	username
+                    			viewCount
```
Comment (moollaza) [inline duckduckhack/instant-answer-display/templates_reference.md]: Looks like there's a tab character lurking in here...

**Comment by moollaza** [comment]
LGTM :+1:

**Pull request merged**